src/bot/boilerplate/messaging.js

```diff
@@ -2,7 +2,7 @@ const { parentPort } = require('worker_threads');
 const __bot = new UserBotClassName();
 const __callbacks = {};
 
-const MESSAGES_PER_SECOND = 20;
+const MESSAGES_PER_SECOND = 50;
```

Comment:
Just a suggestion, my bot was quickly hitting the limit - especially when using`this.log()`